tests/new_upgrades/test_syncplan.py

```diff
@@ -0,0 +1,107 @@
+"""Test for Sync-Plan related Upgrade Scenario's
+
+:Requirement: UpgradedSatellite
+
+:CaseAutomation: Automated
+
+:CaseComponent: SyncPlans
+
+:Team: Phoenix-content
+
+:CaseImportance: High
+
+"""
+
+from datetime import date
+
+from box import Box
+from fauxfactory import gen_alpha, gen_choice
+import pytest
+
+from robottelo.config import settings
+from robottelo.constants import SYNC_INTERVAL
+from robottelo.utils.datafactory import valid_cron_expressions
+from robottelo.utils.shared_resource import SharedResource
+
+
+@pytest.fixture
+def sync_plan_migration_setup(sync_plan_upgrade_shared_satellite, upgrade_action):
```

Comment:
lets change the name to something like 'test_pre_sync_plan_migration_setup' so we are consistent with the preupgrade-postupgrade syntax.